README.rst

```diff
@@ -35,50 +35,15 @@ In order to run Nighres, you will need:
 * Java JDK >= 1.7
 * JCC >= 3.0
 
-For instance, in Debian/Ubuntu (amd64 systems):
-
-    sudo apt-get install openjdk-8-jdk
-    export JCC_JDK=/usr/lib/jvm/java-8-openjdk-amd64
-    python3 -m pip install jcc    (or just pip install jcc if Python 3 is your default)
-
 For some functionalities you need extra packages
 
 * Nipype & ANTs (for registration using ANTs)
 * Pandas (for working with surface meshes)
 * Nilearn (for plotting in the examples)
 
-If you use nighres, please reference the publictation:
-
-Huntenburg JM, Steele CJ, Bazin P-L (2018) Nighres: processing tools for high-resolution neuroimaging. Gigascience 7 Available at: https://academic.oup.com/gigascience/article/7/7/giy082/5049008
-
-Docker
-======
```

Comment:
All docker and install related stuff is now only in the docs: "don't repeat yourself" and "refactoring" is also valid for documentation. :-p